astropy/io/misc/asdf/tags/unit/tests/test_equivalency.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from astropy import units as u
+from astropy.units import equivalencies as eq
+from astropy.cosmology import Planck15
+
+asdf = pytest.importorskip('asdf', minversion='2.3.0.dev0')
+from asdf.tests import helpers
+
+
+equiv = [eq.plate_scale(.3 * u.deg/u.mm), eq.pixel_scale(.5 * u.deg/u.pix),
```

Comment:
This is really just a nit, but I believe that this code gets evaluated every single time that `pytest` runs, which means that it gets evaluated even when we're not running this test (modulo the `importorskip` above). It might make more sense to define this in a function and then call the function in `pytest.mark.parametrize` below.

Reply:
done